src/main.tsx

```diff
@@ -5,6 +5,17 @@ import { ConfigProvider } from 'antd';
 import colors from './colors.module.scss';
 import './global.scss';
 
+import { RootRoute, Route, Router, RouterProvider } from '@tanstack/react-router';
+import { OnboardingPage } from './pages/OnboardingPage';
+
+const rootRoute: RootRoute = new RootRoute({
+    component: App,
+});
+
+const indexRoute = new Route({ getParentRoute: () => rootRoute, path: '/', component: OnboardingPage });
```

Comment:
I think we want to place the routes in a different file like "routes.ts" or something later because otherwise the main.tsx gets overloaded. But we can also move it later.